modules/tests/test_opendaq_device_modules/test_default_config.cpp

```diff
@@ -0,0 +1,240 @@
+#include "test_helpers/test_helpers.h"
+
+using ModulesDefaultConfigTest = testing::Test;
+
+using namespace daq;
+
+TEST_F(ModulesDefaultConfigTest, Create)
+{
+    const auto instance = Instance();
+    const auto config = instance.createDefaultAddDeviceConfig();
+    ASSERT_TRUE(config.hasProperty("general"));
+    ASSERT_TRUE(config.hasProperty("device"));
+    ASSERT_TRUE(config.hasProperty("streaming"));
+}
+
+TEST_F(ModulesDefaultConfigTest, GeneralConfig)
+{
+    const auto instance = Instance();
+    const auto config = instance.createDefaultAddDeviceConfig();
+    const PropertyObjectPtr general = config.getPropertyValue("general");
+    ASSERT_TRUE(general.hasProperty("PrioritizedStreamingProtocols"));
+    ASSERT_TRUE(general.hasProperty("StreamingConnectionHeuristic"));
+}
```

Comment:
The `"AllowedStreamingProtocols"` and `"AutomaticallyConnectStreaming"` properties should likely be tested here as well.